console.lua

```diff
@@ -97,3 +101,107 @@ function BuffDuty:convertPlayerList(identifier, input)
     end
     return result
 end
+
+function Console.parseMessageCommand(cmd, ...)
+    local arg = {...} -- Arg list
+    local idx = 1 -- Current Arg index
+
+    local function arg_valid(i)
+        i = i or idx
+        return (arg[i] and not (i == #arg)) -- Not nil, and ignore the last Arg that AceConsole appends
+    end
+    
+    -- Print Usage Help
+    if arg[1] == "?" or arg[1] == "help" then
+        BuffDuty.printInfoMessage("Usage: /buffduty-msg [options]")
+        BuffDuty.printInfoMessage("reset | Resets all messages to default values")
+        BuffDuty.printInfoMessage("public-title \"value\" | Sets public title message format to \"value\"")
+        BuffDuty.printInfoMessage("duty-line \"value\" | Sets duty line message format to \"value\"")
+        BuffDuty.printInfoMessage("duty-whisper \"value\" | Sets duty whisper message format to \"value\"")
+        BuffDuty.printInfoMessage("single-title \"value\" | Sets single title message format to \"value\"")
+        BuffDuty.printInfoMessage("single-whisper \"value\" | Sets single whisper message format to \"value\"")
+        return false
+    end
+    
+    -- Options
+    while arg_valid() do
+        local tag = string.match(arg[idx], "^[%a%-]+") -- Match starting letters including '-' and '?'
+        -- Reset All
```

Comment:
I have a small suggestion here. What if we add another argument here that would reset specific line?

Reply:
What do you mean? As in resetting for example 'public_title'... this can be done with `/buffduty-msg public-title default`
Or do you think it would be better to have this functionality with `reset`? For example `/buffduty-msg reset all` or `/buffduty-msg reset public-title`

Reply:
Yes, It looks a bit more obvious to me the way with /buffduty-msg reset all or /buffduty-msg reset public-title